Gordon360/Models/ViewModels/ProfileFieldViewModel.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Gordon360.Models.CCT;
+
+namespace Gordon360.Models.ViewModels
+{
+    public class ProfileFieldViewModel
+    {
+        public string field { get; set; }
+        public string value { get; set; }
+        public string label { get; set; }
```

Comment:
In C#, the (widely accepted) convention is that class fields should be capitalized.
```suggestion
        public string Field { get; set; }
        public string Value { get; set; }
        public string Label { get; set; }
```

Reply:
Field is camel case and is used for field identification in Information_Change_Request while label is a formatted string that is used for email sending so that the people at devrequests can see fields like "Personal Email" rather than "personalEmail"

Reply:
That makes sense as far as the difference between Field and Label, thanks.

However, with respect to casing, C# uses PascalCase, not camelCase. That's why I recommended the casing changes.

Reply:
So if you look at any other class, the properties will be PascalCase (i.e. starting with a capital letter). Please capitalize these as well.